src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationIdentificationParametersPresenter.cs

```diff
@@ -114,10 +117,29 @@ public void UpdateStartValueFromSimulation(IdentificationParameterDTO identifica
 
       public void ChangeName(IdentificationParameterDTO identificationParameterDTO, string oldName, string newName)
       {
+         var matchedParameter = _parameterIdentification.AllIdentificationParameters.Single(x => x.Name == oldName);
+         matchedParameter.Name = newName;
+
+         renameParameterAndPublishEvent(oldName, newName);
```

Comment:
I would sooner see this split into
```
if(renameResult(oldName, newName))
  _eventPubliser.PublishEvent(new ParameterIdentificationResultsUpdatedEvent(_parameterIdentification));
```
and just make `renameResult` return true if it changed something